x-pack/plugins/uptime/public/components/monitor_management/monitor_list/enablement_empty_state.tsx

```diff
@@ -102,15 +105,15 @@ const MONITOR_MANAGEMENT_ENABLEMENT_MESSAGE = i18n.translate(
   'xpack.uptime.monitorManagement.emptyState.enablement',
   {
     defaultMessage:
-      'Enable Monitor Management to run lightweight checks and real-browser monitors from hosted testing locations around the world. Enabling Monitor Management will generate an API key to allow the Synthetics Service to write back to your Elasticsearch cluster.',
+      'Enable Monitor Management to run lightweight and real-browser monitors from hosted testing locations around the world. Enabling Monitor Management will generate an API key to allow the Synthetics Service to write back to your Elasticsearch cluster.',
```

Comment:
```suggestion
      'Enable Monitor Management to run lightweight and real-browser monitors from hosted testing locations around the world. Enabling Monitor Management will generate an API key to allow our global managed testing infrastructure to write back to your Elasticsearch cluster.',
```
@drewpost how is this?